core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionFileSystemOutput.java

```diff
@@ -135,11 +135,13 @@ public TransmissionFileSystemOutput(String folderPath) {
     @Override
     public boolean send(Transmission transmission) {
         if (size.get() >= capacityInKB) {
+        	InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.TRACE, "Persistent storage max capcity has been reached; currently at %s KB. Telemetry will be lost, please set the MaxTransmissionStorageFilesCapacityInMB property in the configuration file.", size.get());
             return false;
         }
 
         Optional<File> tempTransmissionFile = createTemporaryFile();
         if (!tempTransmissionFile.isPresent()) {
+        	InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.TRACE, "Persistent storage unable to create temporary file.");
```

Comment:
I think this is an `ERROR` level log. We can still run (i.e. it's not FATAL), but this component cannot do what it's designed to do.

Reply:
I am removing this as it's redundant. There is a logger call in `createTemporaryFile()`

Reply:
Sounds good.